apps/newsletters-api/open-api.yaml

```diff
@@ -184,7 +182,92 @@ paths:
 
 components:
   schemas:
-    # TO DO - define schema for Newsletter when implemented in code.
+    Newsletter:
```

Comment:
Note to self: once PR #59 has been completed, data collected through the wizard can be added to this schema